### PR TITLE
[FAS-11] Add logging and handle exceptions in API routes.

### PR DESCRIPTION
**Pull request by leerob**

### CODE REVIEW

**Comment by leerob** [comment]
#### 📝Changed routes:



- [**/api/feedback**](https://fastfeedback-git-add-logging.leerob1.now.sh/api/feedback)
- [**/api/sites**](https://fastfeedback-git-add-logging.leerob1.now.sh/api/sites)



Commit fe4f625725d2a5f6e3f7266b37989b8fcea28aa8 (https://fastfeedback-git-add-logging.leerob1.now.sh).

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ $ yarn dev
 NEXT_PUBLIC_FIREBASE_API_KEY=
 NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=
 NEXT_PUBLIC_FIREBASE_PROJECT_ID=
+NEXT_PUBLIC_LOGFLARE_KEY=
```
Comment (fenske) [inline README.md:22]: How public are these two really are? 🤔 I briefly Googled but couldn't find any explicit mention.

Comment (leerob) [reply README.md:22]: @chasers I'd love your feedback here. How should I be handling this? I initially made these available on the client-side, but is that the best practice?

Comment (chasers) [reply README.md:22]: For Logflare, the api_key is an ingest api_key which can easily be changed. And the source is the source token (or source ID) which can also easily be changed by adding a new source. Theoretically, someone could spam your logs by making these available on the client but that's the case for Google Analytics, Mixpanel, et al. Neither key allows for any sort of Logflare account management or querying.

Comment (leerob) [reply README.md:22]: Thank you for the info!

**Pull request merged**